### PR TITLE
hatch: Add version 1.9.0

### PR DESCRIPTION
**Pull request by pauleHeisster**
Added new app manifest for hatch.
(Inspired by cruft app manifest)

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/hatch.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            "}",
+            "finally {",
+            "    Remove-Item \"$fname\"",
+            "    deactivate",
+            "    Pop-Location",
+            "}"
+        ]
+    },
+    "bin": "hatch\\Scripts\\hatch.exe",
+    "checkver": "github",
```
Comment (pauleHeisster) [inline bucket/hatch.json:25]:
```suggestion
    "checkver": {
        "github": "https://github.com/pypa/hatch"
    },
```

File: bucket/hatch.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "finally {",
+            "    Remove-Item \"$fname\"",
+            "    deactivate",
+            "    Pop-Location",
+            "}"
+        ]
+    },
+    "bin": "hatch\\Scripts\\hatch.exe",
+    "checkver": {
+        "github": "https://github.com/pypa/hatch"
```
Comment (obfuscated-loop) [inline bucket/hatch.json:26]: Try adding `"regex": "/releases/tag/hatch-(?:v|V)?([\\d\\.]+)"` - I had a similar problem when adding Sonic Visualiser. It's to do with the normal GitHub `checkver` not taking into account that projects sometimes prepend their name to release versions.

Comment (pauleHeisster) [reply bucket/hatch.json:26]:
```suggestion
        "github": "https://github.com/pypa/hatch",
        "regex": "/releases/tag/hatch-(?:v|V)?([\\d\\.]+)"
```

File: bucket/hatch.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "1.7.0",
```
Comment (pauleHeisster) [inline bucket/hatch.json:2]:
```suggestion
    "version": "1.9.0",
```

**Pull request closed without merging**